### PR TITLE
Adding support to devcontainers

### PR DESCRIPTION
**Pull request by thavaahariharangit**
Adding support to devcontainers

Related ticket: https://github.com/github/dependabot-updates/issues/7127

### CODE REVIEW

File: docker/containers.json [hunk 1]
```diff
@@ -16,5 +16,6 @@
   "npm_and_yarn": "ghcr.io/dependabot/dependabot-updater-npm:v2.0.20241001115348@sha256:d7f83d37548b2e77b1552d163d004cb6728705eb1bcb75079eebf26fd1ac1ed3",
   "pip": "ghcr.io/dependabot/dependabot-updater-pip:v2.0.20241001115348@sha256:c3946a2856e3db5aa9b9330ad0ae1e0062413bd4e444fa05717bf0deab71666c",
   "swift": "ghcr.io/dependabot/dependabot-updater-swift:v2.0.20241001115348@sha256:f268d58934a918ec75dcd10597fe0af62442d33b3ca4e378c62d398ee59777ed",
-  "terraform": "ghcr.io/dependabot/dependabot-updater-terraform:v2.0.20241001115348@sha256:dc71828e8198d42bd9641b1c254ab3168b8d3aff27b7e8f6b0133d5efb6777b9"
-}
\ No newline at end of file
+  "terraform": "ghcr.io/dependabot/dependabot-updater-terraform:v2.0.20241001115348@sha256:dc71828e8198d42bd9641b1c254ab3168b8d3aff27b7e8f6b0133d5efb6777b9",
+  "devcontainers": "ghcr.io/dependabot/dependabot-updater-devcontainers:v2.0.20241017235241@sha256:5b854616a508dc1f3e759e0b3ba7abb7ee920797250bbdb2bbb8a1b47a4d43fc"
```
Comment (Nishnha) [inline docker/containers.json:20]: I would have expected the version to match the others. Ist here a resason it's different?

`v2.0.20241001115348` vs `v2.0.20241017235241`

Comment (thavaahariharangit) [reply docker/containers.json:20]: I applied the latest. Thanks, let me update accordingly

**Review by abdulapopoola — APPROVED** [review]

File: docker/Dockerfile.devcontainers [hunk 2]
```diff
@@ -0,0 +1 @@
+FROM ghcr.io/dependabot/dependabot-updater-devcontainers:v2.0.20241017235241@sha256:5b854616a508dc1f3e759e0b3ba7abb7ee920797250bbdb2bbb8a1b47a4d43fc
```
Comment (abdulapopoola) [inline docker/Dockerfile.devcontainers:1]: I think this has to be updated to `v2.0.20241001115348` too?

Comment (thavaahariharangit) [reply docker/Dockerfile.devcontainers:1]: Thanks and updated

**Pull request merged**